roles/contrail_node/tasks/main.yaml

```diff
@@ -0,0 +1,144 @@
+      websslpath: "/etc/contrail"
+      sslsub: "/C=US/ST=CA/LSunnyvale/O=JuniperNetworks/OU=JuniperCA/CN=ContrailCA"
+    run_once: True
+    delegate_to: "{{ groups.lb.0 }}"
+
+  - name: Ensure Contrail WebUI cert path exists
+    file:
+      path: "{{ websslpath }}"
+      recurse: True
+      state: directory
+    run_once: True
+    delegate_to: "{{ groups.lb.0 }}"
+
+  - name: Create SSL certs
+    command: "{{ item }}"
+    with_items:
+      - openssl req -new -newkey rsa:2048 -nodes -out {{ websslpath }}/certrequest.csr -keyout {{ websslpath }}/cs-key.pem -subj {{ sslsub }}
+      - openssl x509 -req -days 730 -in {{ websslpath }}/certrequest.csr -signkey {{ websslpath }}/cs-key.pem -out {{ websslpath }}/cs-cert.pem
+    run_once: True
+    delegate_to: "{{ groups.lb.0 }}"
+
+  - name: Concatenate SSL certs
+    assemble:
+      src: "{{ websslpath }}"
+      dest: "{{ websslpath }}/contrail-webui.pem"
+      remote_src: True
+    run_once: True
+    delegate_to: "{{ groups.lb.0 }}"
+
+  - name: Restart haproxy
+    systemd:
+      name: haproxy
+      state: restarted
+      daemon_reload: yes
+    run_once: True
+    delegate_to: "{{ groups.lb.0 }}"
+
+- when:
+  - loopback.stat.exists == False
+  
+  block:
+  - name: Ensure CNI bin path exists
+    file:
+      path: "/opt/cni/bin"
+      recurse: True
+      state: directory
+      
+  - name: Download & extract loopback CNI binary
+    unarchive:
+      src: "https://github.com/containernetworking/cni/releases/download/{{ cni_version }}/cni-{{ cni_version }}.tgz"
+      dest: "/opt/cni/bin"
+      mode: a+x
+      remote_src: True
+
+- name: Add Contrail service account to privileged scc
+  oc_adm_policy_user:
+    user: system:serviceaccount:kube-system:contrail
+    resource_kind: scc
+    resource_name: privileged
+    state: present
+  run_once: True
+  delegate_to: "{{ contrail_master }}"
+
+- name: Add the daemonset-controller service account to privileged scc
+  oc_adm_policy_user:
+    user: system:serviceaccount:kube-system:daemon-set-controller
+    resource_kind: scc
+    resource_name: privileged
+    state: present
+  run_once: True
+  delegate_to: "{{ contrail_master }}"
+
+- name: Label master nodes with opencontrail.org/controller=true
+  command: oc label nodes {{ hostvars[item]['openshift_hostname'] }} opencontrail.org/controller=true --overwrite=true
+  with_items: "{{ groups.masters }}"
+  run_once: True
+  delegate_to: "{{ contrail_master }}"
+
+- name: Make master schedulable
+  command: oadm manage-node {{ hostvars[item]['openshift_hostname'] }} --schedulable
+  with_items: "{{ groups.masters }}"
+  run_once: True
+  delegate_to: "{{ contrail_master }}"
+
+- name: Launch the Contrail installer
+  command: oc create -f /tmp/contrail-installer.yaml
+  run_once: True
+  delegate_to: "{{ contrail_master }}"
+  ignore_errors: True
+
+- name: Patch restricted scc
+  command: >
+    oc patch scc restricted --patch='{ "runAsUser": { "type": "RunAsAny" } }'
```

Comment:
@sdodson Review this line please. I am not sure if this is acceptable to  patch the scc globally.